.github/workflows/publish.yaml

```diff
@@ -0,0 +1,45 @@
+name: Publish Hub
+
+permissions: read-all
+
+on:
+   workflow_dispatch:
+    inputs:
+      branch:
+        description: "The branch, tag or SHA to checkout"
+        required: true
+        type: string
+        default: "main"
+
+jobs:
+  publish:
+    name: Publish Connector to Hub
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Install Fluvio
+        run: |
+          curl -fsS https://packages.fluvio.io/v1/install.sh | bash
+          echo "$HOME/.fluvio/bin" >> $GITHUB_PATH
+      - name: Install Fluvio CDK
+        run: fluvio install cdk --develop
+      - name: Fluvio Login to Hub
+        run: fluvio cloud login --email ${{ secrets.HUB_USER_EMAIL }} --password ${{ secrets.HUB_USER_PASSWORD }}
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      - name: Install Cross
+        run: cargo install cross
+      - name: Build
+        run: |
+          cross build --release --target aarch64-unknown-linux-musl
+          cp target/aarch64-unknown-linux-musl/release/* target/release/ | true
```

Comment:
this should be put into strategy instead of hard coding target
```
    strategy:
      matrix:
        rust-target: [aarch64-unknown-linux-musl]
        release: [release]
          -  
```

then build can be done as:
```
- name: Build
         run: |
           cross build --release --target ${${{ matrix.rust-target }}
           cp target/${{ matrix.rust-target }}/${{ matrix.release }}/* target/${{ matrix.release }} | true
```

Reply:
Using a strategy matrix will repeat the whole job for a new entry in the matrix. We don't need it because the `cross` tool compiles to the given target in isolation. We don't need to repeat fluvio and cdk installation, checkout, etc. All cross-compilation and publishing will go sequentially, one after another.

Reply:
Fluvio and cdk install is fairly fast.  Most of time will be spent on build which can be paralleled as well as reduce repetition

Reply:
Compilation of `cross` takes a couple of minutes, but parallelism is a good point indeed. Added strategy matrix.